jquery.flot.js

```diff
@@ -1459,14 +1497,12 @@ Licensed under the MIT license.
                 switch(type) {
                     case 'min':
                     case 'max':
-                        // this is a workaround to display the endpoints with a higher
-                        //precision without changing the API and all the tickFormatters
-                        //axis.tickDecimals = (axis.tickDecimals != null ? axis.tickDecimals + 1 : null);
-                        label = axis.tickFormatter(v, axis);
-                        //axis.tickDecimals = (axis.tickDecimals != null ? axis.tickDecimals - 1 : null);
+                        // display endpoints with higer precision
+                        var precision = plot.computeValuePrecision(axis.min, axis.max, axis.direction, axis.options);
```

Comment:
I fail to see how this generates a different precision that the one used for major ticks ?  Can you explain please ?